hkmc2/shared/src/test/mlscript/decls/Prelude.mls

```diff
@@ -36,6 +36,8 @@ declare module Predef with
   class MatchFailure(errors)
   class __EffectSig(next, tail, handleBlockList, resumed, handler, handlerFun)
   class __Return(value)
+  abstract class __StackDelay with
```

Comment:
BTW how come this was not needed before?

Reply:
The code does not rely on the symbol `__StackDelay` being present. But I added it because I want to manually create instance of this class during my benchmark (due to how stack safety works, it only instantiate the effect handler at the top level which it doesn't have access to during the instrumentation of modules)

Reply:
Then yo should say something to that effect in a comment (that this declaration is not really useful except for personal experimentation).